src/netex_idf/stops.rs

```diff
@@ -83,8 +110,101 @@ fn load_stop_areas(elem: &Element) -> Result<CollectionWithId<StopArea>> {
             );
         }
     }
+}
 
-    Ok(stop_areas)
+fn load_coords(quay: &Element) -> Result<(f64, f64)> {
```

Comment:
This function could return a `crate::objects::Coord` instead of a tuple.

Reply:
Mmm, now that I think about it, I think we did mention `Coord` should be only WGS84... which is not the case for these coordinates. So forget about it.